### PR TITLE
:arrow_up: @okta/samples generator 1.15.0

### PR DESCRIPTION
**Pull request by jmelberg-okta**
Updates documentation to prepare for Dev Console rollout.

### CODE REVIEW

File: .samples.config.json [hunk 1]
```diff
@@ -8,7 +8,6 @@
     },
     "server": {
       "staticDir": "dist",
-      "startSignal": "webpack: Compiled successfully",
```
Comment (haishengwu-okta) [inline .samples.config.json]: the start signal is still required by the application server which isn't update to ping the port as start signal.

File: README.md [hunk 2]
```diff
@@ -1,5 +1,3 @@
-[![Build Status](https://travis-ci.org/okta/samples-haskell-scotty.svg?branch=master)](http://travis-ci.org/okta/samples-haskell-scotty/)
```
Comment (haishengwu-okta) [inline README.md]: shall we keep the travis status?

File: README.md [hunk 3]
```diff
@@ -68,23 +66,27 @@ Then, install [stack](https://docs.haskellstack.org/en/stable/README/) and run `
...
-    "oktaUrl": "https://{{yourOktaOrg}}.oktapreview.com",
+    "oktaUrl": "https://{{yourOktaDomain}}.com",
     "clientId": "{{yourClientId}}",
     "clientSecret": "{{yourClientSecret}}",
     "redirectUri": "http://localhost:3000/authorization-code/callback"
   }
 }
 ```
 
-Then, start sample application with `stack exec samples-buildtool -- start --without-mock`
```
Comment (haishengwu-okta) [inline README.md]: shall keep this piece?

File: README.md [hunk 4]
```diff
@@ -268,7 +269,7 @@ ntFBNjluFhNLJIUkEFovEDlfuB4tv_M8BM75celdy3jkpOurg
 ```
 
 ### Validation
-After receiving the `id_token`, we [validate](http://openid.net/specs/openid-connect-core-1_0.html#IDTokenValidation) the token and its claims to prove its integrity.
+After receiving the `id_token`, we [validate](http://openid.net/specs/openid-connect-core-1_0.html#IDTokenValidation) the token and its claims to prove its integrity.
```
Comment (haishengwu-okta) [inline README.md]: nit: space at the end of line.

File: package.json [hunk 5]
```diff
@@ -18,7 +18,7 @@
     "wrap": "node tools/wrap-dependencies",
     "test:mock-okta": "node test/app-server/lib/test-mock-okta",
     "test:app-server": "mocha test/app-server/index --bail --timeout 30000",
-    "test:app-server-all": "mocha test/app-server/with-app-server-start --timeout 900000",
+    "test:app-server-all": "mocha test/app-server/with-app-server-start --timeout 30000",
```
Comment (haishengwu-okta) [inline package.json]: starting the haskell application server is slow (essentially build time). probably need to keep the long timeout.

**Comment by jmelberg-okta** [comment]
@haishengwu-okta Thanks for noticing some of content I failed to include. Updating that now!

**Pull request closed without merging**